pandas/core/computation/parsing.py

```diff
@@ -102,12 +104,12 @@ def clean_column_name(name: str) -> str:
 
     Parameters
     ----------
-    name : str
+    name : hashable
```

Comment:
shouldn't this be Label (or is this consistent with how we do elsewhere?)

Reply:
A quick search reveals 6 counts of "Label", 2 counts of "hashable" (other than the 2 added in this PR). @simonjayhawkins requested hashable over Label, and I like the idea of staying away from pandas-defined types for readability. This is an internal function, so that reasoning doesn't directly apply, but it seems best to me to not have a conditional when deciding on typing in docstrings (e.g. hashable when function is public, Label otherwise).

Reply:
kk that's fine, just worried that we are inconsistent generally. i am not sure this is actually easy to check, but we should try to be as consisten as possible.